src/token/Token.sol

```diff
@@ -403,6 +417,46 @@ contract Token is IToken, UUPS, Ownable, ReentrancyGuard, ERC721Votes, TokenStor
         return super.owner();
     }
 
+    /// @notice Update minters
+    /// @param _minters Array of addresses to add or remove as minters
+    /// @param _isMinter Array of booleans indicating whether the address is a minter
+    function updateMinters(address[] calldata _minters, bool[] calldata _isMinter) external onlyOwner {
+        // Ensure the arrays are the same length
+        if (_minters.length != _isMinter.length) revert INVALID_MINTER_ARRAY_LENGTH();
+
+        // Update each minter
+        for (uint256 i; i < _minters.length; ++i) {
+            // Skip if the minter is already set to the correct value
+            if (minter[_minters[i]] == _isMinter[i]) continue;
+
+            // Update the minter
+            minter[_minters[i]] = _isMinter[i];
+
+            // Emit the event
+            emit MinterUpdated(_minters[i], _isMinter[i]);
```

Comment:
might be a bit cleaner to make this `MintersUpdated(_minters, _isMinter)` and only emit the single event. Gas cost is trivial but I don't know what the advantage of many events would be.

Reply:
+1

Reply:
I think the individual event is preferable from a dumb indexer perspective but thats fine too. I'll change

Reply:
With the individual minter update event, each event is guaranteed to contain new information (see a couple lines above - continue to next array element if minter already has the desired value). The entire array will likely contain redundant info due to people using excess caution and/or not entirely understanding the function.